components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.cassandra/src/main/java/org/wso2/carbon/event/output/adapter/cassandra/CassandraEventAdapter.java

```diff
@@ -178,10 +178,10 @@ public void publish(Object message, Map<String, String> dynamicProperties) {
                 }
                 mutator.execute();
             } catch (Throwable t) {
-                EventAdapterUtil.logAndDrop(eventAdapterConfiguration.getName(), message, null, t, log, tenantId);
+                TenantConfigHolder.logAndDrop(eventAdapterConfiguration.getName(), message, null, t, log, tenantId);
```

Comment:
@ramindu90 

It seems EventAdapterUtil class also had the logAndDrop() method. In that case, if the class is renamed to TenantConfigHolder, keeping the logAndDrop() method inside that class would be wrong. i.e. a call saying TenantConfigHolder.logAndDrop() does not make much sense. Shall we create a new class named EventAdapterUtil and keep that logAndDrop() method there, and keep the getAxisConfig() and addTenantConigContext() methods in TenantConfigHolder class?